frameworks/hdfs/src/main/dist/hdfs-site.xml

```diff
@@ -17,7 +17,12 @@
         <name>dfs.cluster.administrators</name>
         <value>{{ADMINISTRATORS}}</value>
     </property>
-
+    {{#TLS_ENABLED}}
+    <property>
+        <name>dfs.http.policy</name>
+        <value>HTTP_AND_HTTPS</value>
```

Comment:
It can be templated. Other values are:
- HTTP_ONLY
- HTTPS_ONLY
- HTTP_AND_HTTPS